### PR TITLE
Refactor IslCostSpec to test 'unstable isl' behavior

### PR DESCRIPTION
**Pull request by rtretyak**

### CODE REVIEW

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/UnstableIslSpec.groovy [hunk 1]
```diff
@@ -134,24 +82,25 @@ class IslCostSpec extends HealthCheckSpecification {
...
 
-        then: "ISL cost is not increased"
-        swIsls.each { isl ->
-            assert islUtils.getIslInfo(isl).get().cost == swIslsCostMap[isl]
-        }
+        then: "Switch ISL is not 'unstable'"
+        [swIsls[0], swIsls[0].reversed].each { assert database.getIslTimeUnstable(it) == null }
 
         cleanup:
         database.resetCosts()
```
Comment (andriidovhan) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/UnstableIslSpec.groovy]: It seems `resetCosts` is needless here

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/UnstableIslSpec.groovy [hunk 2]
```diff
@@ -95,36 +48,31 @@ class IslCostSpec extends HealthCheckSpecification {
...
             def links = northbound.getAllLinks()
             assert islUtils.getIslInfo(links, isl).get().state == IslChangeType.DISCOVERED
             assert islUtils.getIslInfo(links, isl.reversed).get().state == IslChangeType.DISCOVERED
         }
 
+        then: "Isl is not being 'unstable'"
+        [isl, isl.reversed].each { assert database.getIslTimeUnstable(it) == null }
+
         cleanup:
         database.resetCosts()
```
Comment (andriidovhan) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/UnstableIslSpec.groovy]: `database.resetCosts()` is needless

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/UnstableIslSpec.groovy (hunk 1 above)
Comment (rtretyak) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/UnstableIslSpec.groovy]: yup, looks like you're right. removed

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/UnstableIslSpec.groovy (hunk 2 above)
Comment (rtretyak) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/UnstableIslSpec.groovy]: removed resetCosts

**Comment by surabujin** [comment]
resolve conflicts please

**Pull request merged**